src/components/menuBar/js/menuItemController.js

```diff
@@ -22,11 +22,23 @@ MenuItemController.prototype.init = function(ngModel) {
     this.mode  = $attrs.type;
     this.iconEl = $element[0].children[0];
     this.buttonEl = $element[0].children[1];
-    if (ngModel) this.initClickListeners();
+    if (ngModel) {
+      // Clear ngAria set attributes
+      this.initClickListeners();
+    }
   }
 };
 
+MenuItemController.prototype.clearNgAria = function() {
```

Comment:
Could use a comment here explaining what ngAria does and why it's a problem. (which is my understanding of what's happening here)